codespell_lib/data/dictionary.txt

```diff
@@ -43800,6 +43804,10 @@ pretecting->protecting
 pretection->protection
 pretects->protects
 pretendend->pretended
+pretified->prettified
+pretifies->prettifies
+pretify->prettify
+pretifying->prettifying
```

Comment:
`pretifier->prettifier` and plural?

Also: `prety->pretty`, `pretier->prettier`

Reply:
We already had `prety->pretty, prey,`. I have added `pretier->prettier` and `pretiest->prettiest`.